website/client/src/components/chat/chatMessages.vue

```diff
@@ -315,12 +315,7 @@ export default {
       if (!profile._id) {
```

Comment:
This is probably unrelated to this PR, but since we're here, we're fetching the profile data (cached or not) and then redirecting to the modal page, so the fetched was not used at all, I guess this whole function could be simplified into 

```
async showMemberModal (memberId) {	
    this.$router.push({ name: 'userProfile', params: { userId: memberId } });
}
```